rebar.config.script

```diff
@@ -84,24 +84,5 @@ MaybeFIPSSupport = fun(Config) ->
     end
 end,
 
-MaybeConfigureCoveralls = fun(CONFIG) ->
-    case {os:getenv("GITHUB_ACTIONS"), os:getenv("GITHUB_TOKEN")} of
-        {"true", Token} when is_list(Token) ->
-            CONFIG1 = [{coveralls_repo_token, Token},
-                       {coveralls_service_job_id, os:getenv("GITHUB_RUN_ID")},
-                       {coveralls_commit_sha, os:getenv("GITHUB_SHA")} | CONFIG],
-            case os:getenv("GITHUB_EVENT_NAME") =:= "pull_request" andalso
-                 string:tokens(os:getenv("GITHUB_REF"), "/") of
-                [_, "pull", PRNO, _] ->
-                    [{coveralls_service_pull_request, PRNO} | CONFIG1];
-                _ ->
-                    CONFIG1
-            end;
-        _ ->
-            CONFIG
-    end
-end,
-
-Config0 = MaybeConfigureCoveralls(CONFIG),
-Config1 = SetupIncludedApps(Config0, EnvApps),
-MaybeFIPSSupport(Config1).
```

Comment:
Could you explain in the commit message why it is not needed anymore?

Reply:
done

Reply:
one more note, let's first merge https://github.com/esl/rebar3_codecov/pull/3/files and switch back to the commit version selection for rebar3 codecov plugin before we merge this PR